site/src/components/AppLink/AppLink.tsx

```diff
@@ -43,19 +45,25 @@ export const AppLink: FC<PropsWithChildren<AppLinkProps>> = ({
   health,
 }) => {
   const styles = useStyles()
+  if (appSlug === "") {
+    appSlug = appName
+  }
+  if (appName === "") {
+    appName = appSlug
+  }
```

Comment:
❓What if both `appSlug` and `appName` are empty strings? Is that something we should handle?

Reply:
On the backend this is impossible, one of them will definitely be set.